### PR TITLE
experiment - upgrade py2 -> py3

### PR DESCRIPTION
**Pull request by clarketm**
Additional conversions for python2 -> python3 ( #13164 ) migration and targets the `experiment` directory.

### CODE REVIEW

**Comment by clarketm** [comment]
@spiffxp - With the exception of `generate_tests.py` (which has a build image w/ python3: 4b188e1), I am not aware of any images that are build from any of the files in this PR that were upgraded.

File: experiment/find_developers.py [hunk 1]
```diff
@@ -24,6 +24,7 @@
 import os
 import random
 import sys
+from functools import total_ordering
```
Comment (fejta) [inline experiment/find_developers.py]: Please import modules not methods

http://google.github.io/styleguide/pyguide.html#22-imports

Comment (clarketm) [reply experiment/find_developers.py]: 👍 I will fix this. Also, I will delete this PR and reintroduce the changes *by module* (e.g. `experiment/module`).

**Comment by clarketm** [comment]
Breaking this PR into smaller chunks.
/close

**Pull request closed without merging**